src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color, FigureList.RECTANGLE.name());
+        this.length = length;
+        this.width = width;
+    }
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables for `color` and `name` and pass them on for better code readability.